include/odp/api/spec/crypto.h

```diff
@@ -122,12 +122,26 @@ typedef enum {
 	 */
 	ODP_AUTH_ALG_SHA256_HMAC,
 
+	 /** HMAC-SHA-384
+	 *
+	 *  SHA-384 algorithm in HMAC mode
+	 */
+	ODP_AUTH_ALG_SHA384_HMAC,
+
 	/** HMAC-SHA-512
 	 *
 	 *  SHA-512 algorithm in HMAC mode
 	 */
 	ODP_AUTH_ALG_SHA512_HMAC,
 
+	/** AES-XCBC
+	 *
+	 *  AES-XCBC algorithm, 
+	 *	reference to RFC 3566 
+	 *  The AES-XCBC-MAC-96 Algorithm and Its Use With IPsec
+	 */	
+	ODP_AUTH_ALG_AES_XCBC,
```

Comment:
I'd use AES-XCBC-MAC as the algorithm name and thus enumeration would be ODP_AUTH_ALG_AES_XCBC_MAC. 

Remove reference to IPSEC RFC from here, since this is a crypto API definition, which is more generic than IPSEC use case. Digest size (96) does not need to be mentioned here, as application gets it from capacity call.

In addition to this enum, odp_crypto_auth_algos_t bit field needs to be updated with .aes_xcbc_mac.